benchmarks/cpp/nvfuser/end_to_end.cpp

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
empty line

Reply:
clang-format doesn't complain about it and I normally intentionally add an empty line before the first include.

but if you think it would create any trouble I can remove it.